libs/cardano-ledger-core/src/Cardano/Ledger/Era.hs

```diff
@@ -69,6 +68,25 @@ class
   where
   type Crypto e :: Type
 
+  -- | Extract from TxOut either an address or its compact version by doing the
```

Comment:
Doesn't seem the nicest place for these things?

Reply:
I though about it carefully and I could not find a better place. We already have super constraint on the `Era` that conceptually requires the same functionality: `HasField "address" (Core.TxOut era) (Addr (Crypto era))`, so it only makes sense to have it here.

The only other place I though about placing it under was `UsesTxOut` class in shelley, but I would really like to eventually get rid of the `getField @"address"` hacks in favor of proper API. And for that to work it has to be in the place where `HasField "address"` is required, i.e. `Era` class.